src/styles/blocks/calendar.scss

```
@@ -0,0 +1,64 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+
+  width: $calendarSize;
+  gap: $gap;
+  padding: 10px;
+
+  @each $num, $day in $days {
+    &--start-day--#{$day} :first-child {
```

Maybe this will fix it ;)

```suggestion
    &--start-day--#{$day}:first-child {
```